components/pipeline-service/production/stone-prd-rh01/deploy.yaml

```diff
@@ -0,0 +1,1784 @@
+apiVersion: v1
```

Comment:
1. It would be best if the file started with `---`.
2. This one is a note for later. We'll need to start thinking about automating that soon after this PR is merged. You might want to consider adding a task in the near future so that a PR is raised after the merge to update that file.

Reply:
yes that is doable using infra-deployment task we use in pipeline-service repo. I have already tested it and have a plan for it 🙂